### PR TITLE
Added new section for 2.3 parity

### PR DESCRIPTION
**Pull request by dobooth**
Custom layout updates flow has been changed for 2.3.4 and 2.2.11, here's a guide reflecting those changes

## Purpose of this pull request

This pull request (PR) will guide front-end devs and merchants on the new flow of providing custom layout updates for cms pages, categories and products

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/layouts/xml-manage.html

whatsnew
Added the 'Create cms-page/product/category-specific layouts' section to [Common layout customization tasks](https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/layouts/xml-manage.html).

### CODE REVIEW

File: guides/v2.2/frontend-dev-guide/layouts/xml-manage.md [hunk 1]
```diff
@@ -549,6 +549,42 @@ As result, the `<body>` tag has a new `my-new-body-class` class on all product p
...
+updates are made by creating layout XML files following specific naming conventions.
+
+For Categories:
+
+-  `catalog_category_view_selectable_<Category ID>_<Layout Update Name>.xml`
+
+where:
+
+-  _Category ID_ is desired category ID
+-  _Layout Update Name_ is what is shown as the option for __Custom layout update__ field of __Design__ section on _Category Edit_ page.
```
Comment (shrielenee) [inline guides/v2.2/frontend-dev-guide/layouts/xml-manage.md]: This feels a little inconsistent regarding complete sentences vs. not.

File: guides/v2.2/frontend-dev-guide/layouts/xml-manage.md [hunk 2]
```diff
@@ -549,6 +549,42 @@ As result, the `<body>` tag has a new `my-new-body-class` class on all product p
...
+As of Magento 2.2.11, merchants can select layout updates to be applied to specific Category/Product/CMS Page pages on the frontend. These layout
+updates are made by creating layout XML files following specific naming conventions.
+
+For Categories:
+
+-  `catalog_category_view_selectable_<Category ID>_<Layout Update Name>.xml`
+
+where:
+
+-  _Category ID_ is desired category ID
```
Comment (shrielenee) [inline guides/v2.2/frontend-dev-guide/layouts/xml-manage.md]:
```suggestion
-  _Category ID_ is the desired category ID.
```

File: guides/v2.2/frontend-dev-guide/layouts/xml-manage.md (hunk 1 above)
Comment (shrielenee) [inline guides/v2.2/frontend-dev-guide/layouts/xml-manage.md]:
```suggestion
-  _Layout Update Name_ is shown as the option for the __Custom layout update__ field of the __Design__ section on the _Category Edit_ page.
```

File: guides/v2.2/frontend-dev-guide/layouts/xml-manage.md [hunk 3]
```diff
@@ -549,6 +549,42 @@ As result, the `<body>` tag has a new `my-new-body-class` class on all product p
...
+
+For Products:
+
+-  `catalog_product_view_selectable_<Product SKU>_<Layout Update Name>.xml`
+
+where:
+
+-  _Product SKU_ is the desired product's SKU encoded as a URI.
+  _example_: "My Product SKU" -> "My%20Product%20SKU"
+-  _Layout Update Name_ is what is shown as the option for __Custom layout update__ field of __Design__ section on _Product Edit_ page
```
Comment (shrielenee) [inline guides/v2.2/frontend-dev-guide/layouts/xml-manage.md]:
```suggestion
-  _Layout Update Name_ is shown as the option for the __Custom layout update__ field of the __Design__ section on the _Product Edit_ page.
```

File: guides/v2.2/frontend-dev-guide/layouts/xml-manage.md [hunk 4]
```diff
@@ -549,6 +549,42 @@ As result, the `<body>` tag has a new `my-new-body-class` class on all product p
...
+
+For CMS Pages:
+
+-  `cms_page_view_selectable_<CMS Page Identifier>_<Layout Update Name>.xml`
+
+where:
+
+-  _CMS Page Identifier_ is the desired page's _URL Key_ with "/" symbols replaced with "_"
```
Comment (shrielenee) [inline guides/v2.2/frontend-dev-guide/layouts/xml-manage.md]:
```suggestion
-  _CMS Page Identifier_ is the desired page's _URL Key_ with "/" symbols replaced with "_".
```

File: guides/v2.2/frontend-dev-guide/layouts/xml-manage.md [hunk 5]
```diff
@@ -549,6 +549,42 @@ As result, the `<body>` tag has a new `my-new-body-class` class on all product p
...
+-  _Layout Update Name_ is what is shown as the option for __Custom layout update__ field of __Design__
```
Comment (shrielenee) [inline guides/v2.2/frontend-dev-guide/layouts/xml-manage.md]:
```suggestion
-  _Layout Update Name_ is shown as the option for the __Custom layout update__ field of the __Design__
```

File: guides/v2.2/frontend-dev-guide/layouts/xml-manage.md [hunk 6]
```diff
@@ -549,6 +549,42 @@ As result, the `<body>` tag has a new `my-new-body-class` class on all product p
...
+  section on _CMS Page Edit_ page
```
Comment (shrielenee) [inline guides/v2.2/frontend-dev-guide/layouts/xml-manage.md]: Is this on a separate line than the sentence it is within?

File: guides/v2.2/frontend-dev-guide/layouts/xml-manage.md (hunk 1 above)
Comment (dobooth) [reply guides/v2.2/frontend-dev-guide/layouts/xml-manage.md]: I debate that every time.

File: guides/v2.2/frontend-dev-guide/layouts/xml-manage.md (hunk 6 above)
Comment (dobooth) [reply guides/v2.2/frontend-dev-guide/layouts/xml-manage.md]: unwrapped

**Comment by dobooth** [comment]
running tests

**Pull request merged**